docs/Speech and Audio/Creating Custom Spoken Feedback/index.md

```diff
@@ -0,0 +1,96 @@
+# Creating Custom Spoken Feedback
+Since your user will be driving while interacting with your SDL app, speech phrases can provide important feedback to your user. At any time during your app's lifecycle you can send a speech phrase using the @![iOS]`SDLSpeak`!@@![android,javaSE,javaEE]`Speak`!@ request and SDL's text-to-speech (TTS) engine will produce synthesized speech from your provided text.
+
+When using the `SDLSpeak` RPC, you will receive a response from the head unit once the operation has completed. From the response you will be able to tell if the speech was completed, interrupted, rejected or aborted. It is important to keep in mind that a speech request can interrupt another on-going speech request. If you want to chain speech requests you must wait for for the current speech request to finish before sending the next speech request. 
+
+## Creating the Speak Request
+The speech request you send can simply be a text phrase or it can consist of phoneme specifications to direct SDL’s TTS engine to speak a speech-sculpted phrase. It is also possible to play a pre-recorded sound file using the speech request. For more information on how to play a sound file please refer to [Playing Audio Indications](Speech and Audio/Playing Audio Indications). 
```

Comment:
```suggestion
The speech request you send can simply be a text phrase, which will be played back in accordance with the user's current language settings, or it can consist of phoneme specifications to direct SDL’s TTS engine to speak a language-independent, speech-sculpted phrase. It is also possible to play a pre-recorded sound file (such as an MP3) using the speech request. For more information on how to play a sound file please refer to [Playing Audio Indications](Speech and Audio/Playing Audio Indications). 
```